### PR TITLE
bumper, Add quay login to bumper job

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
when bumping linux-bridge component we need to be registered as a quay user
that has write permissions, in order to push them.
Added a step to login to quay using the "kubevirt+network" quay robot credentials.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/component-bumper-release-0.42.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ jobs:
     - name: Set environment variables
       run: echo "BASE_BRANCH=release-0.42" >> $GITHUB_ENV
 
+    - name: Login to Quay
+      run: docker login -u="kubevirt+network" -p="${{ secrets.QUAY_ROBOT_TOKEN }}" quay.io
```
Comment (qinqon) [inline .github/workflows/component-bumper-release-0.42.yml:18]: Are we allowed to store those tokens here ?

Comment (RamLavi) [reply .github/workflows/component-bumper-release-0.42.yml:18]: why not? the secret is stored in the github repo secret section, and securely used here.

Comment (qinqon) [reply .github/workflows/component-bumper-release-0.42.yml:18]: fair enough

**Pull request merged**